### PR TITLE
Refactor build script. Remove bloat.

### PR DESCRIPTION
**Pull request by ghost**
Removal of the bing chillings. Worked and ran fine from my testing but if I somehow made more bing chilling notify me.

### CODE REVIEW

**Comment by mmvanheusden** [comment]
lao gan ma

**Comment by Avanatiker** [comment]
First of all welcome :smile_cat: Thanks for your contribution.
I'm not sure about some items you are excluding from the jar.
You removed licenses that clearly say to include the copyright notice to any source and binary distribution.

Also regarding the other system files you removed I observed no problems with it yet but i am still not sure how crucial those files were. Can you point out why you removed them and how you knew it doesn't break anything?

**Comment by Avanatiker** [comment]
First of all welcome :smile_cat: Thanks for your contribution.
I'm not sure about some items you are excluding from the jar.
You removed licenses that clearly say to include the copyright notice to any source and binary distribution.

Also regarding the other system files you removed I observed no problems with it yet but i am still not sure how crucial those files were. Can you point out why you removed them and how you knew it doesn't break anything?

**Comment by mmvanheusden** [comment]
I think you should leave the gradlew file as it was for safety reasons

**Comment by ghost** [comment]
> I think you should leave the gradlew file as it was for safety reasons

what

**Comment by ghost** [comment]
> I think you should leave the gradlew file as it was for safety reasons

if ur paranoid about the gradlew file do "./gradlew wrapper --gradle-version=7.4" twice and it updates it to the exact same one as mine if u think I did smth lol

**Comment by Avanatiker** [comment]
The gradlew is untampered I already tested it

File: src/main/kotlin/com/lambda/client/commons/collections/CloseableList.kt [hunk 1]
```diff
@@ -36,15 +36,15 @@ class CloseableList<E>(
 
     override fun removeAll(elements: Collection<E>) =
         if (closed) throw IllegalAccessException("This list is immutable!")
-        else list.removeAll(elements)
+        else list.removeAll(elements.toSet())
```
Comment (Avanatiker) [inline src/main/kotlin/com/lambda/client/commons/collections/CloseableList.kt:39]: Why you do this?

File: src/main/resources/mixins.lambda.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "compatibilityLevel": "JAVA_8",
   "package": "com.lambda.mixin",
   "refmap": "mixins.lambda.refmap.json",
+  "minVersion": "0.8.3",
```
Comment (Avanatiker) [inline src/main/resources/mixins.lambda.json]: Why is this required?

**Comment by ghost** [comment]
Imma close this rq and just make this all into one commit so I don't commit farm

**Comment by NepNep21** [comment]
PRs are merged in a single commit so that doesn't matter anyway :P

**Comment by ghost** [comment]
I forgot that was a feature as my brain is very melted from le scho and gym
